### PR TITLE
Feat/tests

### PR DESCRIPTION
**Pull request by stvnksslr**
not sure if needed to be merged, exploring building out some simple test cases.

### CODE REVIEW

**Comment by jmriebold** [comment]
Looks like this will need to be rebased.

**Comment by stvnksslr** [comment]
Ill fix the conflicts but let me know if these tests make any sense the app itself if simple enough that i was on the fence.

**Comment by jmriebold** [comment]
Yeah I think it's great. Only suggestion I have is, since you're already testing the healthcheck endpoint, you could test the output of /metrics instead of testing `monitorQueue()` directly. There's not a lot of actual logic in `monitorQueue()` anyways.

**Comment by stvnksslr** [comment]
o thats a good point, i didnt think of that since its a big ole text blob

File: main.go [hunk 1]
```diff
@@ -62,15 +67,15 @@ func getMonitorInterval() time.Duration {
...
 
 	monitorInterval, err := strconv.Atoi(monitorIntervalStr)
 	if err != nil {
-		log.Error().Str("errorMessage", err.Error()).Msg("bad value for SQS_MONITOR_INTERVAL")
-		os.Exit(1)
+		log.Warn().Str("errorMessage", err.Error()).Msg("Invalid value for SQS_MONITOR_INTERVAL, using default")
+		return defaultMonitorInterval
 	}
-	return time.Duration(monitorInterval)
+	return time.Duration(monitorInterval) * time.Second
```
Comment (jmriebold) [inline main.go:78]: Oh damn was this a bug?

**Pull request merged**